configure.ac

```diff
@@ -21,7 +21,6 @@ AC_PROG_SED
 
 # Enable silent rules is available
 AM_SILENT_RULES([yes])
-AM_MAINTAINER_MODE([enable])
```

Comment:
Some distributions like to use `--disable-maintainer-mode` to suppress rebuilds, and for those that don't, having the option available but enabled by default is very similar to not having the option. Saving 1 line doesn't seem like a huge benefit.

The upstream project doesn't have generated files in version control, but some downstreams import official source code releases (`make dist` tarballs) into a VCS, in which case all the Autotools cruft is in the VCS.

In the long term, as we gradually drop support for backporting the latest flatpak, xdg-desktop-portal, etc. to ancient distributions, I'd be very tempted to move to Meson (like flatpak/flatpak#2241) which would make this irrelevant.

Reply:
See also #716

Reply:
>  I'd be very tempted to move to Meson

just time for a quick reply today: I have *most* of a meson port of xdg-desktop-portal ready, can push this on monday for discussion

Reply:
I've dropped the patch to remove `AM_MAINTAINER_MODE` now